conda/py38.yaml

```diff
@@ -36,13 +36,15 @@ dependencies:
     - jupyterlab
     - ipython
     - ipykernel
+    - nodejs
 # etc
     - gsw
     - vaex
```

Comment:
Just a comment for everyone's benefit, this is a totally random package that I never ended up using. I just thought it looked cool but ended up not being useful. I would recommend slimming down this yaml file at some point for your own purposes. But, it's a good place to start.